regression/scripts/AVL/avl_files/aircraft.avl

```diff
@@ -0,0 +1,167 @@
+base
```

Comment:
It doesn't look like any of the files in avl_files are used in the regression (and presumably they can't be since we can't run AVL in the regression), so they should be removed.